services/venice-router/src/main/java/com/linkedin/venice/router/api/VenicePathParser.java

```diff
@@ -160,8 +196,17 @@ public VenicePath parseResourceUri(String uri, HTTP_REQUEST request) throws Rout
             pathHelper.getKey(),
             uri,
             partitionFinder,
-            routerStats);
+            routerStats,
+            singleGetRetryManager);
       } else if (VeniceRouterUtils.isHttpPost(method)) {
+        RetryManager multiGetRetryManager = routerMultiGetRetryManagers.computeIfAbsent(
```

Comment:
It seems we would use the same retry manager for both compute and multi-get requests, right?
That would be fine, but we need to call that out explicitly, and rename the variable to be `multiKeyRetryManager`?
If you agree, we need to change all the related occurrences.

Reply:
Right, that's the intention for FC too once we start supporting compute with multiple keys. Will make that change.